src/AppInstallerCommonCore/PathVariable.cpp

```diff
@@ -81,4 +95,10 @@ namespace AppInstaller::Registry::Environment
         std::wstring pathName = std::wstring{ s_PathName };
         m_key.SetValue(pathName, ConvertToUTF16(value), REG_EXPAND_SZ);
     }
+
+    void RefreshPathVariableForCurrentProcess()
+    {
+        const auto& pathValue = PathVariable(Manifest::ScopeEnum::User, true).GetPathValue() + PathVariable(Manifest::ScopeEnum::Machine, true).GetPathValue();

```

Comment:
I think we need to make sure the first value ends with ; before concatenating?

Reply:
Added check to verify value ends with ; before concatenating